lang/ruby/ruby.def

```diff
@@ -0,0 +1,5 @@
+Bootstrap: docker
+From: ruby:2.6-rc-stretch
+
+%runscript
+"$@"
```

Comment:
Def file needs to be still modified, add the help section to describe which version of Ruby is this and also with some instructions on how to use this container, the runscript should be changed to have the ruby command and after that receive parameters  like `ruby ./hello-world.rb`